packages/@aws-cdk/aws-apigatewayv2/lib/websocket/route.ts

```diff
@@ -45,6 +45,12 @@ export interface WebSocketRouteProps extends WebSocketRouteOptions {
    * The key to this route.
    */
   readonly routeKey: string;
+
+  /**
+   * Whether the route requires an API Key to be provided
+   * @default false
+   */
+  readonly apiKeyRequired?: boolean;
```

Comment:
Can this be configured for default, connect and disconnect routes? If so, this should be in `WebSocketRouteOptions`.

Reply:
This can only be configured for the connect route